src/base/QXmppMixIq.h

```diff
@@ -20,7 +21,9 @@ class QXMPP_EXPORT QXmppMixIq : public QXmppIq
         ClientLeave,
         Join,
         Leave,
-        UpdateSubscription,
+#if QXMPP_DEPRECATED_SINCE(1, 7)
+        UpdateSubscription [[deprecated("Use QXmppMixSubscriptionUpdate")]],
```

Comment:
should actually better be `Use QXmppMixManager for this functionality` since the `QXmppMixSubscriptionUpdateIq` is private.